source/replication/site-replication-overview.rst

```diff
@@ -0,0 +1,462 @@
+
+         For example, ``https://<addressforsite>:9000``
+
+      #. Select **Settings**, then **Site Replication**
+      
+      #. Select the trash can Delete icon to the side of the site to update
+
+         .. image:: /images/minio-console/console-site-replication-delete-button.png
+            :width: 600px
+            :alt: MinIO Console's List of Replicated Sites screen with the delete buttons highlighted
+            :align: center
+
+      #. Confirm the site deletion at the prompt by selecting **Delete**
+
+         .. image:: /images/minio-console/console-settings-site-replication-confirm-delete.png
+            :width: 600px
+            :alt: Example of the MinIO Console's Edit Replication Endpoint screen
+            :align: center
+
+   .. tab-item:: Command line
+
+      Use :mc-cmd:`mc admin replicate remove`
+
+      .. code-block:: shell
+
+         mc admin replicate remove <ALIAS> --all --force
+
+      The ``-all`` flag removes the site as a peer from all participating sites.
+
+      The ``--force`` flag is required to removes the site from the site replication configuration.
+
+Review replication status
+-------------------------
+
+MinIO provides information on replication across the sites.
+Information is available in a summary view or for specific, inlcuding users, groups, policies, or buckets.
+
+The summary information includes the number of **Synced** and **Failed** items for each category.
+
+.. tab-set::
+
+   .. tab-item:: Console
+
+      #. In a browser, access the Console for one of the replicated sites
+
+         For example, ``https://<addressforsite>:9000``
+
+      #. Select **Settings**, then **Site Replication**
+      
+      #. Select :guilabel:`Replication Status`
+
+         .. image:: /images/minio-console/console-settings-site-replication-status-summary.png
+            :width: 600px
+            :alt: MinIO Console's Replication status from all Sites screen
+            :align: center
+
+      #. `(Optional)` View the replication status for a specific item
+         
+         Select the type of item to view in the :guilabel:`View Replication Status for a:` dropdown
+
+         Specify the name of the specific Bucket, Group, Policy, or User to view
+
+         .. image:: /images/minio-console/console-settings-site-replication-status-item.png
+            :width: 600px
+            :alt: Example of replication status for a particular bucket item
+            :align: center
+      
+      #. `(Optional)` Update the information by clicking :guilabel:`Refresh`
+
+   .. tab-item:: Command line
+
+      Use :mc-cmd:`mc admin replicate status`
+
+      .. code-block:: shell
+
+         mc admin replicate status <ALIAS> --<flag> <value>
+
+      For example:
+
+      - ``mc admin replicate status minio3 --bucket images``
+
+        Displays the replication status for the ``images`` bucket on the ``minio3`` site.
+        
+        Example output:
```

Comment:
The output resembles the following <- buys us CYA in case the output format changes slightly.

Also as obvious as it should be, we should note what checkmark, x-mark (I assume), and a  blank entry mean